### PR TITLE
Implement sending picture sets by email

### PR DESCRIPTION
**Pull request by ChrisBr**
This implements a feature to send the images directly per mail. Therefore the user needs to go to the gallery page, click send email, enter the email address and click send. See attached images.

E-Mail
image

Gallery:
image

Send email view:
image

I haven't tested this yet on the tablet.

Open questions:
- Move the email page to modal?
- Move email configuration to options?
- Move from field to options?

### CODE REVIEW

File: app/controllers/picture_sets/emails_controller.rb [hunk 1]
```diff
@@ -1,6 +1,5 @@
 module PictureSets
   class EmailsController < ApplicationController
-    protect_from_forgery with: :exception
```
Comment (ChrisBr) [inline app/controllers/picture_sets/emails_controller.rb]: Why did you remove it here?

**Comment by digitaltom** [comment]
i18n works well :+1:

**Comment by ChrisBr** [comment]
@digitaltom from my side it is ready to merge :+1:
